_data/navigation.yml

```diff
@@ -26,6 +26,10 @@ docs:
         url: /docs/body-rules
       - title: 'Policies'
         url: /docs/policies
+  - title: 'Orbital response rules'
```

Comment:
Can we rename this to `Orbital response types`

Reply:
Fixed